src/lib/api/upload-blob.ts

```diff
@@ -74,7 +74,7 @@ async function withSafeFile<T>(
       return await fn(newPath)
     } finally {
       // Remove the temporary file
-      await RNFS.unlink(newPath)
+      await deleteAsync(newPath)
```

Comment:
does this need to be `safeDeleteAsync` instead?

Reply:
not in this case since we've already passed a good uri to `withSafeFile` (from above in `uploadBlob` where we prepend `file://` if needed)

ill use it though here just to promote the use of `safeDelete`